.bmad-core/templates/project-brief-tmpl.yaml

```
@@ -0,0 +1,221 @@
+      - "Analyze resource/constraint trade-offs"
+      - "Generate risk mitigation strategies"
+      - "Challenge scope from MVP minimalist view"
+      - "Brainstorm creative feature possibilities"
+      - "If only we had [resource/capability/time]..."
+      - "Proceed to next section"
+
+sections:
+  - id: introduction
+    instruction: |
+      This template guides creation of a comprehensive Project Brief that serves as the foundational input for product development.
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)